dokka-subprojects/plugin-base-frontend/build.gradle.kts

```diff
@@ -21,18 +22,36 @@ node {
 
 val distributionDirectory = layout.projectDirectory.dir("dist")
 
+tasks.npmInstall {
+    // enable caching - workaround for https://github.com/node-gradle/gradle-node-plugin/issues/81
+    outputs.file(layout.projectDirectory.file("node_modules/.package-lock.json"))
+        .withPropertyName("nodeModulesPackageLock")
+    outputs.cacheIf { true }
+}
```

Comment:
This doesn't work as I expected, and additionally causes a problem because the NpmInstall task [already has a `nodeModulesPackageLock` output property](https://github.com/node-gradle/gradle-node-plugin/blob/c07b5ea719b757eabb4e3076741a2d069cc50a2d/src/main/kotlin/com/github/gradle/node/npm/task/NpmInstallTask.kt#L110) which clashes with this task (although sometimes it is null, so I think it only clashes sometimes?).


Ideally I'd want Gradle to skip the npmInstall task if npmRunBuild is UP-TO-DATE or FROM-CACHE, but I think that's something that needs to change in the Gradle Node Plugin.

Instead, I propose removing `node_modules/` from the clean task. That should improve performance locally.